src/main/java/primeNumbers/PrimeNumbersMain.java

```diff
@@ -0,0 +1,20 @@
+package primeNumbers;
+
+import org.slf4j.*;
+
+import java.util.List;
+
+public class PrimeNumbersMain {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PrimeNumbersMain.class);
+
+    public static void main(String[] args) {
+        int to = 50;
+
+        PrimeNumbersDefinition definePrimeNumbers = new PrimeNumbersDefinition();
+        List<Integer> list = definePrimeNumbers.getPrimeNumbers(to);
+
+        for (int i : list) {
+            LOGGER.info(i + "");
+        }
```

Comment:
Let's try to rewrite it in a functional way

Reply:
Done in e10c88a644d8e317f6eebafd5b061a0e6df590d7.
'for' loop is now changed to .stream()... method.